roles/install/meta/argument_specs.yml

```diff
@@ -51,3 +51,10 @@ argument_specs:
         description:
           - 'Whether the role should use C(become: true) when installing packages.'
         type: bool
+      sops_github_token:
+        description:
+          - Token to provide when querying the GitHub API for the latest release. Without the token
+            there are rather strict rate limits.
+          - Should mainly be used in GitHub Actions.
+        type: string
```

Comment:
This is what caused the weird string conversion error, argument_spec expects `type: str`